source/sync/data-model/data-model-map.txt

```diff
@@ -22,12 +22,14 @@ Overview
 --------
 
 This page contains information on how the App Services Schema used by
-Atlas Device Sync maps to the Realm Object Schema used by the Realm SDKs.
+Atlas Device Sync maps to the Realm Object Schema used by the Atlas Device SDKs.
 
-To generate Realm SDK Schemas from your App Services Schemas,
-refer to :ref:`Generate SDK Object Models <generate-sdk-object-models>`.
-To generate App Services Schemas from Realm SDK client code,
-refer to :ref:`Development Mode <development-mode>`.
+To learn how to generate these schemas, refer to the following pages:
+
+- To generate Realm Object Schemas from your App Services Schemas, refer to :ref:`Generate SDK Object Models <generate-sdk-object-models>`.
+- To generate App Services Schemas from Atlas Device SDK client code, refer to
+  :ref:`Generate App Services Schema from a Realm Object Model
```

Comment:
In cases like this, I wish we could refer to the "Realm Object Model" as the "Atlas Device SDK Object Model" for consistency. But the naming will do for now.